modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -49,39 +49,28 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}}{{^parent}}{{#ha
   @SerializedName("{{baseName}}")
   {{/gson}}
   {{#isContainer}}
-  {{#useBeanValidation}}@Valid{{/useBeanValidation}}
-  {{#openApiNullable}}
-  private {{>nullableDataType}} {{name}} = {{#isNullable}}JsonNullable.undefined(){{/isNullable}}{{^isNullable}}{{#required}}{{{defaultValue}}}{{/required}}{{^required}}null{{/required}}{{/isNullable}};
-  {{/openApiNullable}}
-  {{^openApiNullable}}
-  private {{>nullableDataType}} {{name}} = {{#required}}{{{defaultValue}}}{{/required}}{{^required}}null{{/required}};
-  {{/openApiNullable}}
+  {{#useBeanValidation}}
+  @Valid
+  {{/useBeanValidation}}
   {{/isContainer}}
-  {{^isContainer}}
   {{#isDate}}
   @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
   {{/isDate}}
   {{#isDateTime}}
   @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
   {{/isDateTime}}
-  {{#openApiNullable}}
-  private {{>nullableDataType}} {{name}}{{#isNullable}} = JsonNullable.undefined(){{/isNullable}}{{^isNullable}}{{#defaultValue}} = {{{.}}}{{/defaultValue}}{{/isNullable}};
-  {{/openApiNullable}}
-  {{^openApiNullable}}
-  private {{>nullableDataType}} {{name}}{{#isNullable}} = null{{/isNullable}}{{^isNullable}}{{#defaultValue}} = {{{.}}}{{/defaultValue}}{{/isNullable}};
-  {{/openApiNullable}}
-  {{/isContainer}}
+  private {{>dataType}} {{name}}{{#defaultValue}} = {{{defaultValue}}}{{/defaultValue}};
```

Comment:
Worth noting: This changes the default way that containers were working before (defaulted to `null` if not required)